lib/ffmpeg/vaapi/vpp.py

```diff
@@ -42,6 +42,14 @@ def gen_vpp_opts(self):
     elif self.vpp_op in ["overlay"]:
       vpfilter.append("[1:v]hwupload[v1];")
       vpfilter[-1] += "[0:v][v1]overlay_vaapi=alpha={a}".format(a = mapRange(self.alpha, [0.0, 255.0], [0.0, 1.0]))
+    elif self.vpp_op in ["pad"]:
+      vpfilter.append("pad_vaapi")
+      if hasattr(self, "padw") and hasattr(self, "padh"):
+        vpfilter[-1] += f"=width={self.padw}:height={self.padh}"
+        if hasattr(self, "x") and hasattr(self, "y"):
```

Comment:
We have to have both `x` and `y`?  Why not allow just `x` or just `y`?

Reply:
There are no dependency between `x` and `y`,  I'll follow your suggestion to update it.

Reply:
Updated, thx